devicon.json

```
@@ -8945,5 +8945,21 @@
         },
         "color": "#F04E98",
         "aliases": []
+    },
+    {
+      "name": "githubactions",
+      "altnames": [
+          "GitHub Actions"
+      ],
+      "tags": [
+          "devops",
+          "integration"
+      ], 
+      "versions": {
+        "svg": ["original", "plain"],
+        "font": ["plain"]
+      },
```

Align `versions` properties arrays.
```suggestion
      "versions": {
        "svg": [
            "original",
            "plain"
        ],
        "font": [
            "plain"
        ]
      },
```